src/foundry/client/data/documents/cards.d.mts

```diff
@@ -160,19 +172,24 @@ declare global {
      */
     get canClone(): boolean;
 
-    // TODO: Figure out the typing here
     static override createDocuments(
-      data: Array<ConstructorDataType<typeof Cards> | (ConstructorDataType<typeof Cards> & Record<string, unknown>)>,
-      context: DocumentModificationContext & { temporary: false },
+      data: Array<
+        | fields.SchemaField.AssignmentType<Cards["schema"]["fields"]>
```

Comment:
Isn't this just `foundry.documents.BaseCards.ConstructorData`

Reply:
It looks like it's `foundry.documents.BaseCards.UpdateData`